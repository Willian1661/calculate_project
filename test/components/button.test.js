const button = require("../../src/components/button")

describe("Pattern button component", () => {

	it("has a html button's  tag", () =>{
		expect(button("foo","bar","is-baz")).toMatchSnapshot();
	});
	
});