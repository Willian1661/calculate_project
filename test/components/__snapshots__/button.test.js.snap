// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pattern button component has a html button's  tag 1`] = `"<button class="button is-baz" data-value="bar">foo</button>"`;
